site/index.html

```diff
@@ -1,15 +1 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>A website!</title>
-    <link rel="stylesheet" href="css/main.css">
-  </head>
-  <body>
-    <header id="page-header">
-      <h1>Amazing Photo Gallery</h1>
-    </header>
-    <p>
-      Coming soon™.
-    </p>
-  </body>
-</html>
+<img src="images/photo_1040.jpg" height="400" width="550"><img src="images/photo_1068.jpg" height="400"width="550" ><img src="images/photo_134.jpg" height="400"width="550" ><img src="images/photo_173.jpg" height="400"width="550" ><img src="images/photo_215.jpg" height="400"width="550" ><img src="images/photo_220.jpg" height="400"width="550" ><img src="images/photo_221.jpg" height="400"width="550" ><img src="images/photo_261.jpg" height="400"width="550" ><img src="images/photo_31.jpg" height="400"width="550" ><img src="images/photo_443.jpg" height="400"width="550" ><img src="images/photo_474.jpg" height="400"width="550" ><img src="images/photo_525.jpg" height="400"width="550" ><img src="images/photo_600.jpg" height="400"width="550" ><img src="images/photo_612.jpg" height="400"width="550" ><img src="images/photo_69.jpg" height="400"width="550" ><img src="images/photo_719.jpg" height="400"width="550" ><img src="images/photo_819.jpg" height="400"width="550" ><img src="images/photo_949.jpg" height="400"width="550" >
```

Comment:
You can see that the enclosing HTML is gone, including the link to the stylesheet.